investpy/stocks.py

```diff
@@ -1506,3 +1506,176 @@ def search_stocks(by, value):
     search_result.reset_index(drop=True, inplace=True)
 
     return search_result
+
+
+def get_stock_earnings(stock, country):
```

Comment:
As there are some changes in the `stocks.py` file in the master branch, could you pull the current file from the master and add your function at the end so that the PR has no merge conflicts?